nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -125,6 +125,12 @@
       <link linkend="opt-networking.interfaces">networking.interfaces.&lt;name&gt;.…</link> options.
     </para>
    </listitem>
+   <listitem>
+     <para>
+       The option <literal>environment.homeBinInPath</literal> has been introduced and defaulted to <literal>false</literal>.
```

Comment:
So the change in 20.03 is that `~/bin` will no longer be in path, it's important to users that did rely on this to know to enable this option. Can our release note explain this?

Reply:
Perhaps  if we just flip it around a bit. But it looks good :+1: 

```
Going forward, ~/bin/ in the users home directory will no longer be in PATH by default.
If you depend on this you should set the option environment.homeBinInPath to true.
The aforementioned option was added this release.
```

Reply:
I with your version :) Thanks!